### PR TITLE
prefill cc from draft if set

### LINKED ISSUE #619: cc is not filling in from draft
#613 tried to fix cc problem but it didn't fix the whole thing.
I think `value="{{replyCc}}"` was introduced from filling in cc from draft, but the problem is that `replyCc` is not a list. I think to fix it down to the bottom we need to make draft in local storage to store CC in list, then use is directly, or parse what is stored to make it a list, then fill it in.

### PR DESCRIPTION
**Pull request by wurstchristoph**
fixes #619

@clementhk

### CODE REVIEW

**Comment by clementhk** [comment]
Ok I would prefer to cache cc as list so it uses the method to print, but this should work.

Review-by: me.

File: js/mail.js [hunk 1]
```diff
@@ -292,6 +292,22 @@ var Mail = {
 				return str;
 			});
 
+			Handlebars.registerHelper("ifHasCC", function (cc, ccList, options) {
+				if ((cc !== undefined) || (ccList !== undefined)) {
```
Comment (DeepDiver1975) [inline js/mail.js]: I prefer the use of _.isUndefined() - compare more reliably

**Comment by wurstchristoph** [comment]
@DeepDiver1975 updated my PR. Please review again ;-)

**Comment by jancborchardt** [comment]
@clementhk is this now cool from your POV as well? And @DeepDiver1975?

**Comment by DeepDiver1975** [comment]
:+1:

**Pull request merged**